helm/cluster/templates/_helpers.tpl

```diff
@@ -305,6 +305,34 @@ fake-app-catalog-from-offline-cluster-chart-rendering
 {{- end }}
 {{- end }}
 
+{{/*
+  cluster.app.in-release is a public named helper template that checks if the app that is specified under
+  property 'appName' in the object that is passed to the template exists in the Release.
+
+  Example usage in template:
+
+    {{- $_ := set $ "appName" "foo-bar-controller" }}
+    {{- $appInRelease := include "cluster.app.in-release" $ }}
+    {{- if $appInRelease }}
+    {{!-- Do logic here... --}}
+    {{- end }}
+*/}}
+{{- define "cluster.app.in-release" }}
+{{- $_ := (include "cluster.internal.get-release-resource" $) }}
+{{- $inRelease := false }}
+{{- $renderWithoutReleaseResource := ((($.GiantSwarm.internal).ephemeralConfiguration).offlineTesting).renderWithoutReleaseResource | default false }}
+{{- if $renderWithoutReleaseResource }}
+{{- $inRelease = $renderWithoutReleaseResource }}
```

Comment:
```suggestion
{{- $inRelease = true }}
```

I know, they are logically the same, but I'd assign `true` for readability reasons. And maybe because of side effects.